packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.ts

```diff
@@ -480,21 +572,9 @@ export class AtomicSearchInterface
     window.addEventListener('hashchange', this.onHashChange);
   }
 
-  private initAriaLive() {
-    if (
-      Array.from(this.host.children).some(
-        (element) => element.tagName === 'ATOMIC-ARIA-LIVE'
-      )
-    ) {
-      return;
-    }
-    const ariaLive = document.createElement('atomic-aria-live');
-    this.host.prepend(ariaLive);
-  }
-
   private initRelevanceInspector() {
-    if (this.enableRelevanceInspector) {
-      this.host.addEventListener('dblclick', (e) => {
+    if (this.enableRelevanceInspector && !this.disableRelevanceInspector) {
+      this.addEventListener('dblclick', (e) => {
```

Comment:
Currently we're not removing this listener when the component is removed from the DOM.